src/Microsoft.Extensions.Configuration.AzureAppConfiguration/LogHelper.cs

```diff
@@ -74,5 +74,10 @@ public static string BuildPushNotificationUnregisteredEndpointMessage(string res
         {
             return $"{LoggingConstants.PushNotificationUnregisteredEndpoint} '{resourceUri}'.";
         }
+
+        public static string BuildFailoverToDifferentEndpointMessage(string originalEndpoint, string currentEndpoint)
+        {
+            return $"{LoggingConstants.RefreshFailoverToDifferentEndpoint} Original Endpoint:'{originalEndpoint.TrimEnd('/')}' Current Endpoint:'{currentEndpoint.TrimEnd('/')}'";
```

Comment:
Not sure about having both the original and current endpoint, if it seems unnecessary I can change it to being just the current one.

Reply:
I think it makes sense to log previous and next endpoints. I'm imagining logs like this:

`Failed to get configuration settings from 'contoso.azconfig.io'. Trying to read from 'contoso-replica1.azconfig.io'`

`Failed to get configuration settings from 'contoso-replica1.azconfig.io'. Trying to read from 'contoso-replica2.azconfig.io'`

Reply:
Makes sense, do you think I should remove the final log saying what endpoint succeeded then or keep it along with these? I think it just clarifies which endpoint succeeded, but I can also understand that the last log of the ones above could imply that the last endpoint worked.

Reply:
We dont need warning level logs for successful refresh operation.

Reply:
Oh didn't think of that way, sounds good